packages/globlon/globlon.1.0/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A globbing library for OCaml"
+description:
+  "A globbing library for OCaml. We use it to match file paths against patterns."
+maintainer: ["bleepbloopisfy"]
+authors: ["bleepbloopisfy"]
```

Comment:
any chance you could add an email address for people who don’t use github?

Reply:
done, added the email address and left the existing username